### PR TITLE
authentication policy contract data source

### PR DESCRIPTION
**Pull request by wesleymccollam**

### CODE REVIEW

File: internal/resource/config/administrativeaccount/administrative_account_data_source.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ var (
 )
 
 // Create a Administrative Account data source
-func NewAdministrativeAccountDataSource() datasource.DataSource {
+func AdministrativeAccountDataSource() datasource.DataSource {
```
Comment (henryrecker-pingidentity) [inline internal/resource/config/administrativeaccount/administrative_account_data_source.go:22]: Why are we changing all these?

File: internal/resource/config/authenticationpolicycontract/authentication_policy_contract_data_source.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+// authenticationPolicyContractDataSource is the datasource implementation.
+type authenticationPolicyContractDataSource struct {
+	providerConfig internaltypes.ProviderConfiguration
+	apiClient      *client.APIClient
+}
+
+// GetSchema defines the schema for the datasource.
+func (r *authenticationPolicyContractDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	schema := schema.Schema{
+		Description: "Manages an authentication policy contract.",
```
Comment (henryrecker-pingidentity) [inline internal/resource/config/authenticationpolicycontract/authentication_policy_contract_data_source.go:34]: Should be "describes" for a data source

File: internal/resource/config/authenticationpolicycontract/authentication_policy_contracts_common.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package authenticationpolicycontract
```
Comment (henryrecker-pingidentity) [inline internal/resource/config/authenticationpolicycontract/authentication_policy_contracts_common.go:1]: I like moving this stuff to a common file

**Review by henryrecker-pingidentity — CHANGES_REQUESTED** [review]

File: internal/resource/config/administrativeaccount/administrative_account_data_source.go (hunk 1 above)
Comment (wesleymccollam) [reply internal/resource/config/administrativeaccount/administrative_account_data_source.go:22]: to match the resource format - no need to have "New"

File: internal/resource/config/authenticationpolicycontract/authentication_policy_contract_data_source.go (hunk 2 above)
Comment (wesleymccollam) [reply internal/resource/config/authenticationpolicycontract/authentication_policy_contract_data_source.go]: updated

**Pull request merged**